src/Components/Scenes/Hydration/NewJob/NewJob.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Box } from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import NewJobHeader from './NewJobHeader';
+
+const chromeStyles = makeStyles(theme => ({
+  container: {
+    position: 'fixed',
+    zIndex: 1101, // Positions container on top of original header bar
+    height: '100%',
+    width: '100%',
+    display: 'flex'
+  },
+  // Placeholder pattern until canvas is implemented
+  canvas: {
+    background:
+      'repeating-linear-gradient(45deg, #EDF5FF, #EDF5FF 15px, #FFFFFF 5px, #FFFFFF 20px)',
+    flexGrow: 1
+  },
+  toolbar: {
+    minWidth: '350px',
+    backgroundColor: 'white'
+  }
+}));
+
+const HydrationChrome = props => {
+  const classes = chromeStyles();
+
+  return (
+    <Box className={classes.container}>
+      <Box className={classes.canvas}>
+        <NewJobHeader jobName="My New Job"></NewJobHeader>
+      </Box>
+      {/* Placeholder box until toolbar is implemented */}
```

Comment:
Nitpicky thing on any placeholders (and I'm realizing I could do this in a couple places too) - at what level of severity do we want to add and track something like `TODO`s?

Almost more of an open question for the team. Thanks for adding comments to clarify in general!

Reply:
On remote branches I think anything goes, but to keep the main codebase clean, I'm a big fan of not merging any placeholder content or incomplete functions. My experience is, things generally get forgotten about much more quickly once they've been merged, advancing codebase entropy.

Reply:
I'd be happy to throw a bunch of TODO's at this to clarify what still needs to be fleshed out. I tend to feel that without the placeholders though, PRs will get too huge when trying to implement these components in one fell swoop. Since the Hydration UI is essentially one of our highest priorities and we will continue work on it immediately after, it doesn't bother me in this case.

Reply:
Another note, this layout might also be changing drastically once I start exploring with the project storm react-diagrams and how interactions between the toolbar and canvas will look. I figured this at least creates a place where I can continue with that spike work and see what will work best.